editors/document-model/editor-schema.tsx

```diff
@@ -136,8 +139,11 @@ export default function EditorSchema({
         codegen(code).then(result => {
             import(/* @vite-ignore */ result).then(validators => {
                 const schemaName = `${name}StateSchema`;
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
```

Comment:
it seems like the types of this code are out of our control, so I suppressed these rules here and let them stay as `any`